lisa/testsuite.py

```diff
@@ -237,10 +237,26 @@ class TestCaseRequirement:
     os_type: Optional[search_space.SetSpace[Type[OperatingSystem]]] = None
 
 
-def simple_requirement(
+def _construct_node(
+    node: Optional[schema.NodeSpace] = None,
     min_count: int = 1,
     min_nic_count: int = 1,
-    node: Optional[schema.NodeSpace] = None,
+    min_data_disk_count: int = 0,
+    data_disk_caching_type: str = constants.DATADISK_CACHING_TYPE_NONE,
+    data_disk_iops: int = 120,
+) -> schema.NodeSpace:
+    if node is None:
+        node = schema.NodeSpace()
+        node.node_count = search_space.IntRange(min=min_count)
+        node.nic_count = search_space.IntRange(min=min_nic_count)
+        node.data_disk_count = search_space.IntRange(min=min_data_disk_count)
+        node.data_disk_caching_type = data_disk_caching_type
+        node.data_disk_iops = data_disk_iops
+    return node
+
+
+def _construct_test_case_requirement(
```

Comment:
"create" is more popular than "construct" here.

Reply:
fix